Anlab.Mvc/Client/components/PaymentSelection.tsx

```diff
@@ -1,115 +1,127 @@
+                      className={!isUcClient ? activeDiv : inactiveDiv}
+                      onClick={() => this._handleChange("creditcard")}
+                    >
+                        <h3>Credit Card</h3>
+                        <p>It's amazing what you can do with a little love in your heart.</p>
+                    </div>
+                    <span className="dividing_span col-2 t-center align-middle">or</span>
+                    <div
+                      className={isUcClient ? activeDiv : inactiveDiv}
+                      onClick={() => this._handleChange("uc")}
+                    >
+                        <h3>UC Funds</h3>
+                        <p>It's amazing what you can do with a little love in your heart</p>
+                    </div>
+                </div>
+                {this._renderUcAccount()}
+            </div>
+        );
     }
-    _renderUcAccount = () => {
-        if (this.props.payment.clientType === 'uc') {
+
+    private _renderUcAccount = () => {
+        if (this.props.payment.clientType === "uc") {
             return (
                 <div>
-                    <Input type="text" label="UC Account" error={this.state.error} value={this.props.payment.account} maxLength={50} onChange={this.handleAccountChange} onBlur={this.lookupAccount} />
+                    <Input
+                      type="text"
+                      label="UC Account"
+                      error={this.state.error}
+                      value={this.props.payment.account}
+                      maxLength={50}
+                      onChange={this._handleAccountChange}
+                      onBlur={this._lookupAccount}
+                    />
                     {this.state.accountName}
                 </div>
             );
         }
     }
 
-    checkChart = (chart: string) => {
-        if (chart === "L" || chart === "l" || chart === "3")
-            return true;
-        else
-            return false;
+    private _checkChart = (chart: string) => {
+        return (chart === "L" || chart === "l" || chart === "3") ;
     }
 
-    lookupAccount = () => {
-        if (!this.state.error && this.props.payment.account !== null && this.checkChart(this.props.payment.account.charAt(0)))
-        {
-            fetch(`/financial/info?account=${this.props.payment.account}`, { credentials: 'same-origin' })
-                .then(response => {
-                    if (!response.ok) {
-                        throw Error();
-                    }
-                    return response;
-                })
-                .then(response => response.json())
-                .then(accountName => this.setState({ accountName }))
-                .catch(error => {
-                    this.setState({ accountName: null, error: "The account you entered could not be found" });
-                });
-        }
-        else
-        {
-            this.setState({ accountName: null });
+    private _lookupAccount = () => {
```

Comment:
@jpknoll There is a problem here clearing out the error message when it isn't a UC Chart (3 or L is UC)